### PR TITLE
Enable SA1504: All accessors should be single-line or multi-line

### PR DESCRIPTION
**Pull request by xtqqczze**
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1504.md

### CODE REVIEW

**Comment by xtqqczze** [comment]
Depends on #14243.

File: src/Microsoft.Management.Infrastructure.CimCmdlets/GetCimAssociatedInstanceCommand.cs [hunk 1]
```diff
@@ -90,7 +90,10 @@ public string ResultClassName
         [Alias(CimBaseCommand.AliasCimInstance)]
         public CimInstance InputObject
         {
-            get { return cimInstance; }
+            get
+            {
+                return cimInstance;
+            }
```
Comment (iSazonov) [inline src/Microsoft.Management.Infrastructure.CimCmdlets/GetCimAssociatedInstanceCommand.cs:96]: It is auto fix? Is it possible to use modern `get => cimInstance;`?

Comment (xtqqczze) [reply src/Microsoft.Management.Infrastructure.CimCmdlets/GetCimAssociatedInstanceCommand.cs:96]: There is an open issue for expression-bodied accessors: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/2402

Comment (xtqqczze) [reply src/Microsoft.Management.Infrastructure.CimCmdlets/GetCimAssociatedInstanceCommand.cs:96]: `get => cimInstance;` is not currently a violation, but may not be in the spirit of the rule.

**Review by iSazonov — APPROVED** [review]

**Pull request merged**